annotations/testdata/augmenter-input-multiple-predicates-per-concept.json

```diff
@@ -3,6 +3,10 @@
 		"predicate": "http://www.ft.com/ontology/annotation/about",
 		"id": "http://www.ft.com/thing/b224ad07-c818-3ad6-94af-a4d351dbb619"
 	},
+	{
```

Comment:
It is invalid to have mentions and about for the same concept. Why is this added?

Reply:
For test purposes, to check with more than 2 elements in the array.
Ok, I can change the predicate.